jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -62,6 +63,19 @@ public Builder setTargetImage(@Nullable ImageReference imageReference) {
       return this;
     }
 
+    public Builder setBaseAndTargetImage(String baseImage, String targetImage)
+        throws InvalidImageReferenceException {
+      baseImageReference = ImageReference.parse(baseImage);
+      targetImageReference = ImageReference.parse(targetImage);
+      if (baseImageReference.usesDefaultTag()) {
```

Comment:
Error messages should generally go in the `build` method since setters are intended to be called multiple times.